### ISSUE TITLE
WMT14en2de validation dataset is not newstest2013?

### ISSUE THREAD

**Issue by haorannlp** [pytorch/fairseq#2559]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
I understand that ``prepare-wmt14en2de.sh`` generates train/valid/test sets automatically, but I do not see ``newstest2013``, which is used as dev set in many papers, is directly used. The valid set is instead generated by splitting the training set? Do I miss something here or I should modify the script to account for ``newtest2013``? 
If we are using ``newstest 2013`` as valid set, ``newstest 2014`` as test set. The best BLEU score on the dev set should be close to the score on the test set. I trained a Transformer base on europarlv7, it can score over 30 BLEU score on the dev set, but only 20 on the test set? This might suggest that actual dev set we use is not newstest 2013? Thank you
#### What's your environment?

 - fairseq Version (e.g., 1.0 or master): 0.9
 - PyTorch Version (e.g., 1.0): 1.2
 - OS (e.g., Linux): Linux
 - How you installed fairseq (`pip`, source): source
 - Build command you used (if compiling from source):
 - Python version: 3.7
 - CUDA/cuDNN version: 10.2
 - GPU models and configuration:
 - Any other relevant information:

**Comment by haorannlp** [comment]
@myleott Hi, could you look at this problem? I'm really in a hurry.

**Comment by haorannlp** [comment]
@alexeib Hi, could you look at this query? Am I missing something here?

**Comment by edunov** [comment]
Hi @haorannlp, 

We used a subset extracted from train as validation set in this setup, that was a fairly common practice back when this script was written. You're right that nowadays, I'd use newstest13 for validation instead. If you'd like, feel free to add an option to the script to use newstest2013 as a valid and send a pull request. 

Thanks,
Sergey
